spec/views/events/show.html.erb_spec.rb

```diff
@@ -211,6 +211,24 @@
     expect(rendered).to_not have_link(t(:show_participants, scope: 'events.participants'))
   end
 
+  it "does not display send acceptances button after acceptances have been sent in selection phase" do
+    @event = assign(:event, FactoryGirl.create(:event, :in_selection_phase))
```

Comment:
> that could happen at some point in the future, but imo its not relevant yet, because the current implementation checks whether a mail was sent, not if there are receivers waiting for one.

You are right, for any reason I thought that it is not possible to send acceptance mails if there are no accpeted applicants. But currently there isn't a check like that so it is fine